client/post-editor/media-modal/gallery/edit.jsx

```diff
@@ -40,24 +41,44 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { site, settings } = this.props;
+		const { onUpdateSetting, site, settings, translate } = this.props;
 
 		if ( ! site || ! settings.items ) {
 			return null;
 		}
 
+		const orders = {
+			[ translate( 'Reverse order' ) ]: reverse( [ ...settings.items ] ),
+			[ translate( 'Order alphabetically' ) ]: sortBy( settings.items, 'title' ),
+			[ translate( 'Order chronologically' ) ]: sortBy( settings.items, 'date' ),
```

Comment:
This shows oldest media first. Is that what one expects from "order chronologically"?